webpack/redux/reducers/helpers.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/prefer-default-export
+export function parseMissingPermissions(errMsg) {
+  const defaultMsg = ['unknown'];
+  try {
+    // 1. "Missing one of the required permissions: view_subscriptions, edit_manifest"
+    // 2. ["Missing one of the required permissions", "view_subscriptions, edit_manifest"]
+    // 3. ["view_subscriptions", "edit_manifest"]
+    const missingPermissions = errMsg.split(': ')[1].split(', ');
```

Comment:
I'm not sure if this convention exists in foreman and its plugins

Reply:
@amirfefer you're right, it's definitely hacky.  I think we should just have the Foreman API give us the array we need.